src/Cdn/Cdn/Models/Endpoint/PSDeliveryRule.cs

```diff
@@ -20,11 +20,13 @@ namespace Microsoft.Azure.Commands.Cdn.Models.Endpoint
 {
     public class PSDeliveryRule
     {
+        public string Name{ get; set; }
+        
         public int Order { get; set; }
 
-        public IList<PSDeliveryRuleAction> Actions { get; set; }
```

Comment:
@hytao do you need to rename the `Actions` and `Conditions` properties in this file? We should be able to keep them plural since they aren't parameters, and making this change would be a breaking change if the `PSDeliveryRule` type was exposed to the user (_e.g._, as an output or parameter type)